### ISSUE TITLE
voc trainning convergence problem

### ISSUE THREAD

**Issue by vsooda** [zhreshold/mxnet-yolo#3]
@zhreshold thank you for your great job.

I use your `dev` branch to train voc. After 206 batch, the results is: 

```
INFO:root:Epoch[206] Train-Acc=0.084337
INFO:root:Epoch[206] Train-IOU=0.387760
INFO:root:Epoch[206] Train-BG-score=0.002908
INFO:root:Epoch[206] Train-Obj-score=0.003026
INFO:root:Epoch[206] Time cost=391.103
INFO:root:Saved checkpoint to "/home/sooda/deep/object/mxnet-yolo/model/yolo2_416-0207.params"
INFO:root:Epoch[206] Validation-aeroplane=0.000000
INFO:root:Epoch[206] Validation-bicycle=0.000000
INFO:root:Epoch[206] Validation-bird=0.000000
INFO:root:Epoch[206] Validation-boat=0.000000
INFO:root:Epoch[206] Validation-bottle=0.000000
INFO:root:Epoch[206] Validation-bus=0.000000
INFO:root:Epoch[206] Validation-car=0.000000
INFO:root:Epoch[206] Validation-cat=0.000000
INFO:root:Epoch[206] Validation-chair=0.000000
INFO:root:Epoch[206] Validation-cow=0.000000
INFO:root:Epoch[206] Validation-diningtable=0.000000
INFO:root:Epoch[206] Validation-dog=0.001082
INFO:root:Epoch[206] Validation-horse=0.000000
INFO:root:Epoch[206] Validation-motorbike=0.000000
INFO:root:Epoch[206] Validation-person=0.004746
INFO:root:Epoch[206] Validation-pottedplant=0.000000
INFO:root:Epoch[206] Validation-sheep=0.000000
INFO:root:Epoch[206] Validation-sofa=0.000000
INFO:root:Epoch[206] Validation-train=0.000000
INFO:root:Epoch[206] Validation-tvmonitor=0.000000
INFO:root:Epoch[206] Validation-mAP=0.000291
```
Maybe something is wrong?  your `master` branch is ok?

I also try to use your `master` branch, but when I run `git submodule update`, it reports `Unable to checkout '9b9855bae145f0a387accf94ca36687e6fe4ee53' in submodule path 'mxnet'`, maybe you need to update your master branch's submodule?

**Comment by zhreshold** [comment]
try dev branch with lr=0.0001

**Comment by vsooda** [comment]
@zhreshold  thank you very much. I change the learning rate as you said. It is much better.

I didn't use pretrain model, and after 200 epoch, mAP is 0.51.

```
INFO:root:Epoch[200] Train-Acc=0.882000
INFO:root:Epoch[200] Train-IOU=0.685558
INFO:root:Epoch[200] Train-BG-score=0.004245
INFO:root:Epoch[200] Train-Obj-score=0.256506
INFO:root:Epoch[200] Time cost=297.076
INFO:root:Saved checkpoint to "/home/sooda/deep/object/mxnet-yolo/model/yolo2_416-0201.params"
INFO:root:Epoch[200] Validation-aeroplane=0.595343
INFO:root:Epoch[200] Validation-bicycle=0.642839
INFO:root:Epoch[200] Validation-bird=0.428441
INFO:root:Epoch[200] Validation-boat=0.413425
INFO:root:Epoch[200] Validation-bottle=0.204297
INFO:root:Epoch[200] Validation-bus=0.623591
INFO:root:Epoch[200] Validation-car=0.685286
INFO:root:Epoch[200] Validation-cat=0.599618
INFO:root:Epoch[200] Validation-chair=0.307342
INFO:root:Epoch[200] Validation-cow=0.450331
INFO:root:Epoch[200] Validation-diningtable=0.528938
INFO:root:Epoch[200] Validation-dog=0.531340
INFO:root:Epoch[200] Validation-horse=0.639920
INFO:root:Epoch[200] Validation-motorbike=0.583562
INFO:root:Epoch[200] Validation-person=0.632997
INFO:root:Epoch[200] Validation-pottedplant=0.254757
INFO:root:Epoch[200] Validation-sheep=0.497984
INFO:root:Epoch[200] Validation-sofa=0.472311
INFO:root:Epoch[200] Validation-train=0.686695
INFO:root:Epoch[200] Validation-tvmonitor=0.527394
INFO:root:Epoch[200] Validation-mAP=0.515321
```

**Comment by vsooda** [comment]
@zhreshold after 240 epochs, the mAP drop to 0.48. the highest mAP is 0.52. I only use one gpu. how can I get a better result? use a pretrained model?

Thank you.

**Comment by zhreshold** [comment]
A pretrained model on imagenet is prefered

**Comment by vsooda** [comment]
Thanks, I will have a try.
